extension-manifest-v2/src/modules/autoconsent.js

```diff
@@ -0,0 +1,99 @@
+/**
```

Comment:
how about calling the folder background?

Reply:
The `src` is already a `background` in your sence. Usually, there are classes, so there is `src/classes` path, but I would avoid that pattern with self initialized class (it should be es module). This is why I named it `modules`.